docs/index.md

```diff
@@ -0,0 +1,157 @@
+  - list_item: Image annotation and review.
+  - list_item: Run algorithms on all or parts of images.
+
+# Resource links
+about_resources:
+  - name: Gitter Channel
+    link: #
+    icon: assets/img/resource-icons/icon_gitter.png
+
+  - name: Discourse Forum
+    link: #
+    icon: assets/img/resource-icons/icon_discourse.png
+
+  - name: JupyterNotebooks
+    link: #
+    icon: assets/img/resource-icons/icon_jupyternb.png
+
+  - name: Youtube Channel
+    link: #
+    icon: assets/img/resource-icons/icon_youtube.png
+
+# Platforms Content
+# =================================
+platforms:
+  # HistomicsUI
+  - title: HistomicsUI
+    description: Web-based application for examining, annotating, and processing histology images to extract both low and high level features (e.g. cellular structure, feature types).
+    docs_url: google.com
+    github_url: github.com
+    screenshot: assets/img/histomicsui_screenshot.png
+    features:
+      - name: Secure Data Management
+        description: The DSA provides fine-grained user or role-based access to datasets, images & metadata, and annotations. Amazon S3 hosting supported.
+        icon: assets/img/feature-icons/icon-secure_data_management.svg
+
+      - name: RESTful APIs
+        description: A rich API allows programmatic control over users, data, annotations, and algorithms, enabling automation of DSA tasks and integration with other tools and platforms.
+        icon: assets/img/feature-icons/icon-restful_apis.svg
+
+      - name: Visualization and Annotation
+        description: An optimized user interface provides fluid exploration of large whole-slide images and tools for efficient generation of image markups.
+        icon: assets/img/feature-icons/icon-visualization_annotation.svg
+
+      - name: Execution Engine
+        description: Girder provides distributed execution and monitoring of algorithm and analytics jobs.
+        icon: assets/img/feature-icons/icon-execution_engine.svg
+
+      - name: Broad Support for Histology Image Formats
+        description: A wide variety of tiled image formats are supported, including tiff, svs, and jp2. Images can be retiled automatically as needed for processing algorithms. Additional formats can be added with a pluggable Python interface.
+        icon: assets/img/feature-icons/icon-history_image_formats-primary.svg
+
+  # HistomicsTK
+  - title: HistomicsTK
+    description: HistomicsTK is a Python toolkit based on image processing, for quantitative analysis of whole-slide digital pathology images.
+    docs_url: google.com
+    github_url: github.com
+    screenshot: assets/img/histomicstk_screenshot.png
+    features:
+      - name: Preprocessing and Format Support
+        description: HistomicsTK provides color normalization and deconvolution operations to improve the robustness of analytic pipelines. The Large Image plugin gives programmers a convenient interface for reading from whole-slide imaging formats.
```

Comment:
Since we mention formats in greater detail below, lets change this section to just `Preprocessing` and remove the "The Large Image plugin..." sentence.

Reply:
fixed in commit 51f03b5f5d244cd83cc6acb83fbdc3cba4987b7d